onnx_tf/handlers/backend/conv_mixin.py

```diff
@@ -93,9 +95,14 @@ def conv(cls, node, input_dict, transpose=False):
     group = node.attrs.get("group", 1)
     weight_shape = weights.get_shape().as_list()
     # Is this convolution depthwise we can support?
-    depthwise = (x_rank == 4 and len(weight_shape) == 4 and
+    if isinstance(x_shape, np.ndarray):
+      depthwise = (x_rank == 4 and len(weight_shape) == 4 and
                 group != 1 and group == x_shape[1] and not transpose and
                 not (None in weight_shape))
+    else:
+      depthwise = (x_rank == 4 and len(weight_shape) == 4 and
```

Comment:
To avoid copying the full bool condition you may consider doing it this way
```
depthwise = (x_rank == 4 and len(weight_shape) == 4 and
                group != 1 and not transpose and
                not (None in weight_shape))
if isinstance(x_shape, np.ndarray):
    depthwise = (depthwise and group == x_shape[1])
```

Reply:
Code is simplified, same logic.